platform/linux-generic/include/odp_chksum_internal.h

```diff
@@ -0,0 +1,201 @@
+/* Copyright (c) 2020, Nokia
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:	BSD-3-Clause
+ */
+
+/**
+ * @file
+ *
```

Comment:
Only API files (and maybe some examples) generate doxygen documentation. Implementation internal files do not need doxygen tags (and /**). You can test with "make doxygen-doc" that there's no warnings after removing tags.

Reply:
That's not a reason to not use doxygen comments, though. I usually use doxygen comments just to conform to a known format. Whether some documentation is actually generated from it is a different matter. Also, there are plenty of doxygen comments in linux-gen.

So the question isn't whether it's needed, but whether it's allowed.

Reply:
I agree that the format is known, but in general we should avoid adding unnecessary lines or code that is not tested by CI. When our build does not run doxygen on these internal files, tags will be broken during future edits of these files. When people tend to create new files by copying the old ones, broken tags will spread further, and in the end there would be many files that look like having proper doxygen documentation, but if you run doxygen on those the end result would be  errors and potentially no useful documentation.

Usually, I remove extra doxygen tags from internal files while editing those.